torch/_torch_docs.py

```diff
@@ -662,6 +706,28 @@ def merge_dicts(*dicts):
     tensor([ 0.9833,  0.0811, -1.9743, -1.4151])
 """.format(**common_args))
 
+add_docstr(torch.atanh,
+           r"""
+atanh(input, out=None) -> Tensor
+
+Returns a new tensor with arctanh (inverse hyperbolic tangent) of the elements of :attr:`input`.
+
+.. math::
+    \text{out}_{i} = \tanh^{-1}(\text{input}_{i})
+""" + r"""
+Args:
+    {input}
```

Comment:
Needs description.

Reply:
I've added the description, but left a comment here: https://github.com/pytorch/pytorch/pull/38388#discussion_r426151237. Thanks!